website/docs/usage-config.md

```diff
@@ -201,10 +201,11 @@ from typing import Iterable
 import thinc
 
 @thinc.registry.schedules("my_cool_decaying_schedule.v1")
-def decaying(base_rate: float, decay: float, *, t: int = 0) -> Iterable[float]:
-    while True:
-        yield base_rate * (1.0 / (1.0 + decay * t))
-        t += 1
+def decaying(base_rate: float, decay: float, *, t: int = 0) -> SchedulerCallable:
```

Comment:
Same change as the one in `schedules.py`.

Reply:
Fixed.